### PR TITLE
Update J. League.md

### PR DESCRIPTION
**Pull request by nayananandofficial**
Updated Top 5 J league Goal Poachers/Advanced Strikers.

### CODE REVIEW

**Comment by nayananandofficial** [comment]
Inspired by your project, I wanted to contribute something from my side. If I have your permission, I would be pleased to contribute more as a developer/efootball player.

**Comment by rahaaatul** [comment]
> Inspired by your project, I wanted to contribute something from my side. If I have your permission, I would be pleased to contribute more as a developer/efootball player.

Yeah, absolutely! You don't need my permission. It's open source so anyone can contribute.

I stopped updating since there's not enough player that care about this stuff, so I was discouraged to do it.

File: data/top-five/J. League.md
Comment (rahaaatul) [inline data/top-five/J. League.md:1]: Can you capitalize players names, please? 

Example: `Anderson Lopes` instead of `anderson lopes`.

**Comment by nayananandofficial** [comment]
Thank you for the response Rahatul. I have a craze for this stuff such as
analysing football data and live updating them. My goal was to build
something similar or bigger than efootballhub and pesdb. Later I understood
that's a challenging task and can take a long time to build. So I searched
for some repositories and found yours.

I'll make the changes that you have recommended. Thank you.

**Comment by nayananandofficial** [comment]
I'll try to contribute more. Looking forward to it!

**Pull request merged**